doc/adrs/208_add_edge_type_in_style.adoc

```diff
@@ -0,0 +1,48 @@
+= ADR-208 - Support edge type in edge style
+
+== Context
+
+Today, Sirius Web offers two types of edges: rectangular (default) and smart edge (which prevents edges from crossing over a node).
+
+[source, typescript]
+----
+export const useEdgeType = (): UseEdgeTypeValue => {
+  const [edgeType, setEdgeType] = useState<DiagramEdgeType>('smoothStepEdge');
+  return {
+    edgeType,
+    setEdgeType,
+  };
+};
+----
+
+You can switch between the two types in the panel bar of a diagram, but the change applies to the entire diagram.
+The idea is to be able to specify directly in the edge style which type to apply and thus be able to change it edge by edge using the appearance palette.
+
+== Decision
+
+Add a new feature in edgeStyle
+
+[source]
+----
+<eClassifiers xsi:type="ecore:EEnum" name="EdgeType">
+    <eLiterals name="Manhattan"/>
+    <eLiterals name="Smart Manhattan" value="1"/>
+  </eClassifiers>
+<eStructuralFeatures xsi:type="ecore:EAttribute" name="edgeType" lowerBound="1"
+        eType="#//EdgeType" defaultValueLiteral="InputArrow"/>
```

Comment:
`defaultValueLiteral="InputArrow"` I don't believe this part, I suspect that it should be `defaultValueLiteral="Manhattan"`